tests/h/subscribers_test.py

```diff
@@ -80,13 +80,14 @@ def test_calls_get_notification_with_request_annotation_and_action(self, fetch_a
         event = AnnotationEvent(mock.sentinel.request,
                                 {'id': mock.sentinel.annotation_id},
                                 mock.sentinel.action)
+        mock.sentinel.request.db = mock.Mock()
```

Comment:
This reads a bit oddly to me. Probably better to have `request = mock.Mock(db=mock.sentinel.session)`